yarn.lock

```diff
@@ -2508,6 +2508,70 @@
     mkdirp "^0.5.1"
     rimraf "^2.5.2"
 
+"@material-ui/core@^4.9.7":
```

Comment:
@lychyi I'm not sure why this dependency was added.

Reply:
good catch, not sure why any of these are added. I'll checkout yarn.lock from master and try again